### DISCUSSION TITLE
Ultima Theme - Misc and overlay pages with error, running on Mojarra 2.2

### DISCUSSION THREAD

**Discussion by erickdeoliveiraleal** [primefaces/community#2365] [PrimeFaces Templates] [answered]
When running on Mojarra 2.2 the misc and overlay pages generates errors: java.lang.IllegalStateException: Nested form with ID 'j_idt104:j_idt110' encountered inside parent form with ID 'j_idt104'. This is illegal in HTML.

**Comment by melloware** [comment, marked as answer]
yep you can't nest a form inside a form.  Here are those two pages fixed with extra `h:form` removed.
[ultima-misc-overlay-fixed.zip](https://github.com/user-attachments/files/16324456/ultima-misc-overlay-fixed.zip)
